client/my-sites/sharing/connections/service-action.jsx

```diff
@@ -73,7 +73,7 @@ class SharingServiceAction extends Component {
 				primary={ primary }
 				borderless={ false }
 				scary={ warning }
-				compact={ true }
+				compact
 				onClick={ this.onActionClick }
 				disabled={ isPending }>
```

Comment:
Also, note that React is clever enough to remove this native attribute when it's false.